src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -119,7 +119,7 @@
   </resheader>
   <data name="ColorScheme_Background.Text" xml:space="preserve">
     <value>Background</value>
-    <comment>The background color for text displayed on the screen. Users can select a color to use in this context.</comment>
+    <comment>This is the header for a control that lets the user select the background color for text displayed on the screen.</comment>

```

Comment:
i was really hoping you would apply this to all of the color comments that were phrased the same way... >_<

Reply:
derp hahaha. Onnit